ha/setup/ha_setup.py

```diff
@@ -777,6 +777,15 @@ def process(self):
         Process init command.
         """
         Log.info("Processing init command")
+        env_type = self.get_installation_type().lower()
+        if env_type == const.INSTALLATION_TYPE.HW.value.lower():
+            Log.info("Enabling the stonith.")
+            self._execute.run_cmd(const.PCS_STONITH_ENABLE)
+            Log.info("Stonith enabled successfully.")
+        elif env_type == const.INSTALLATION_TYPE.VM.value.lower():
+            Log.warn("Stonith configuration not available, detected VM env")
```

Comment:
VM

f"Stonith configuration not available, detected {env_type} env"

Reply:
DONe